include/flashinfer/sampling.cuh

```diff
@@ -1166,6 +1168,7 @@ __global__ void ChainSpeculativeSampling(DType* draft_probs, IdType* draft_token
       SamplingTempStorage<DType, BLOCK_THREADS, SCAN_ALGORITHM, REDUCE_ALGORITHM>&>(smem_sampling);
 
   uint32_t pos = 0;
+  uint32_t stop_pos = num_speculative_tokens;
   for (pos = 0; pos < num_speculative_tokens; ++pos) {
```

Comment:
I prefer rename the iteration variable to `i`, which is consistent with the later loop.